components/segmented-control/src/segmented-control.js

```diff
@@ -35,7 +35,7 @@ export const SegmentedControl = ({
     return (
         <ul className="segmented-control" aria-label={ariaLabel}>
             {options.map(({ label, value, disabled }) => (
-                <li key="label">
+                <li key={`option-${value}`}>
```

Comment:
I guess the original intent was to use the label as the key, but forgot to use the variable.

Value makes sense though!